cltk/tokenize/word.py

```diff
@@ -18,7 +18,7 @@ def __init__(self, language):
         """Take language as argument to the class. Check availability and
         setup class variables."""
         self.language = language
-        self.available_languages = ['latin']
+        self.available_languages = ['latin','sanskrit','hindi']
```

Comment:
This change I made keeping the flow in mind, as I saw word tokenizer is calling the underlining functions so I made the change to make it appear encapsulated.